index.bs

```diff
@@ -2698,6 +2707,72 @@ To <dfn>get the navigation info</dfn>, given |context| and |navigation status|:
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.captureScreenshot Command ####  {#command-browsingContext-captureScreenshot}
+
+The <dfn export for=commands>browsingContext.captureScreenshot</dfn> command
+captures an image of the given browsing context, and returns it as a
+Base64-encoded string.
+
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextCaptureScreenshotCommand = {
+        method: "browsingContext.captureScreenshot",
+        params: BrowsingContextCaptureScreenshotParameters
+      }
+
+      BrowsingContextCaptureScreenshotParameters = {
+        context: BrowsingContext
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+        BrowsingContextCaptureScreenshotResult = {
+          data: text
+        }
+    </pre>
+   </dd>
+</dl>
+
+TODO: Full page screenshots, and multiple output formats
```

Comment:
This comment is confusing. I understand "document element's rectangle" as the full-page. Do I misunderstand "full-page" or "document element's rectangle"?

Reply:
https://w3c.github.io/webdriver/#dfn-draw-a-bounding-box-from-the-framebuffer is limiting the screenshot to the viewport, irrespective of the rectangle dimensions.